### DISCUSSION TITLE
Long intros break the page styling in "for you" section

### DISCUSSION THREAD

**Discussion by m3rashid** [community/community#17031] [Other Feature Feedback, Questions, & Ideas]
![Screenshot_20220521-091902](https://user-images.githubusercontent.com/64685317/169634303-96e56390-5745-4e18-8755-77f309788b36.png)
Page styling is distorted 

(Posted in discussions coz I couldn't find the issues section) 🥲

**Comment by edhends** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
